apps/deno/tests/test1.ts

```diff
@@ -98,6 +98,8 @@ export default async (assets: Assets) => {
       xSkew: degrees(0),
```

Comment:
These are good integration test updates. However, it's important that we test all page drawing methods that the dash array and dash phase options can be applied to. Right now we're only testing `PDFPage.drawCircle` and `PDFPage.drawLine`.

Reply:
I've just added drawRectangle, drawEllipse, and drawSquare with (border)DashArray in addition. And we already had one for drawSvgPath within test12. I think we cover all cases now